application/src/main/java/org/togetherjava/tjbot/commands/moderation/BanCommand.java

```diff
@@ -89,7 +89,7 @@ private static RestAction<Boolean> sendDm(ISnowflake target,
         String dmMessage =
                 """
                         Hey there, sorry to tell you but unfortunately you have been banned %s from the server %s.
-                        If you think this was a mistake, please contact a moderator or admin of the server.
+                        To get in touch with a moderator, you can simply use the **/modmail** command here in this chat. Your message will then be forwarded and a moderator will get back to you soon :thumbsup:
```

Comment:
Because most of our members have no experience with using slash commands, they have a lot of issues using them correctly. So to help them out, we started migrating all `/command` strings to [slash command mentions](https://discord.com/developers/docs/change-log#slash-command-mentions).

Issue: https://github.com/Together-Java/TJ-Bot/issues/559
PR: https://github.com/Together-Java/TJ-Bot/pull/565

So I would place a TODO mentioning this, so we don't forget about it. If that PR gets merged before this one, it's us to align with it.